test/e2e/specs/editor/various/multi-block-selection.spec.js

```diff
@@ -1089,7 +1089,11 @@ test.describe( 'Multi-block selection (@firefox, @webkit)', () => {
 
 		await page.keyboard.press( 'ArrowLeft' );
 		// Select everything between [].
-		await pageUtils.pressKeys( 'Shift+ArrowLeft', { times: 5 } );
+		// Delay ensures selection can catch up.
+		await pageUtils.pressKeys( 'Shift+ArrowLeft', {
+			times: 5,
+			delay: 50,
+		} );
```

Comment:
It looks like Playwright just got faster, so I had to add delays to ensure text selection can catch up before performing actions 😅

I don't think a similar bug can be reproduced manually by humans.